### ISSUE TITLE
`no-unused-svelte-ignore` false positive for a11y warnings on elements with inner element

### ISSUE THREAD

**Issue by DetachHead** [ota-meshi/eslint-plugin-svelte#386]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.35.0

### What version of `eslint-plugin-svelte` are you using?

2.19.1

### What did you do?

<details>
<summary>Configuration</summary>

```
plugin:svelte/recommended
```
</details>

```svelte
<!-- svelte-ignore a11y-no-noninteractive-tabindex -->
<span tabindex="0">
    <span class="element" />
</span>
```

### What did you expect to happen?

no error, as the compile warning is being suppressed by the `svelte-ignore` comment

### What actually happened?

```
[1:20]: svelte-ignore comment is used, but not warned (svelte/no-unused-svelte-ignore)
[2:1]: A11y: noninteractive element cannot have nonnegative tabIndex value(a11y-no-noninteractive-tabindex) (svelte/valid-compile)
```

### Link to **GitHub Repo** with Minimal Reproducible Example

https://ota-meshi.github.io/eslint-plugin-svelte/playground/#eJw1y8EKgzAQhOFX2e59sV7F+iS5rHEogXRTzCKW4rsbKYU5fT/z5VgW8MDjTYTqhuyQ9LSygrTvP2KlzZI5Vo2eNojrnGzBTiJTsLG+1ehvj8D3wI2JfiFmrbUpMl4wD0zddequOPFxAsAxKvI=

### Additional comments

the `selte/valid-compile` warning appears even though the `svelte-ignore` comment successfully suppresses the warning when running `svelte-check`

**Comment by DetachHead** [comment]
another example
```svelte
<!-- svelte-ignore a11y-click-events-have-key-events -->
<div on:click={() => {}}>
  <span class="element" />
</div>
```

**Comment by ota-meshi** [comment]
Thank you for the bug report. I will fix that.
